scripts/scil_tractogram_fix_trk.py

```diff
@@ -128,69 +134,72 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
-    warning_msg = """
-        # This script is still experimental, DSI-Studio and Startrack
-        # evolve quickly and results may vary depending on the data itself
-        # as well as DSI-studio/Startrack version.
-    """
-
-    logging.warning(warning_msg)
-
+    logging.warning("This script is not fully tested, DSI-Studio and "
+                    "Startrack evolve quickly and results may vary depending "
+                    "on the data itself as well as DSI-studio/Startrack "
+                    "version.")
     assert_outputs_exist(parser, args, args.out_tractogram)
 
     if args.software == 'startrack':
+        if (args.in_dsi_fa or args.in_native_fa or args.auto_crop or
+                args.save_transfo or args.load_transfo):
+            parser.error("For the startrack software, please only set the "
+                         "--reference option.")
+        if args.reference is None:
+            parser.error("For the startrack software, please set the "
+                         "--reference option.")
         assert_inputs_exist(parser, [args.in_tractogram, args.reference])
         sft = load_tractogram(args.in_tractogram, 'same',
                               bbox_valid_check=args.bbox_check,
                               trk_header_check=False)
-        new_sft = StatefulTractogram(sft.streamlines, args.reference,
-                                     Space.VOX)
+        sft = StatefulTractogram(sft.streamlines, args.reference, Space.VOX)
 
         # Startrack flips the TRK
         flip_axis = ['x']
-        new_sft.to_vox()
-        new_sft.streamlines._data -= get_axis_flip_vector(flip_axis)
-        new_sft = flip_sft(new_sft, flip_axis)
-        new_sft.to_rasmm()
-
-    else:
+        sft.to_vox()
+        sft.streamlines._data -= get_axis_flip_vector(flip_axis)  # --------------------_> HEin?
```

Comment:
@frheault can you help with this one please ?

Reply:
+ L185

Reply:
I dont have data from StarTrack, but since there is a flip in X for everything we cannot use 'to_corner()' or 'to_center()' since it would shift it in the opposite direction. I believe this was the equivalent of centering the streamlines to the voxel before flipping.

DSI is flip in X/Y (LPS instead of RAS), so the shift is made to center streamlines to the voxel before the flip.